.gitlab/source_test/linux.yml

```diff
@@ -70,6 +71,7 @@
     - source /root/.bashrc && conda activate ddpy3
     - inv -e rtloader.make --install-prefix=$CI_PROJECT_DIR/dev --python-runtimes "3"
     - inv -e rtloader.install
+    - inv -e sds.build-library
```

Comment:
Are we sure we need this ? I'm never sure when mixing compilation / shared libs with linting (which is supposed to be before compilation)

Reply:
Might be unnecessary for the linting indeed, I'll look into this.